clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/mark_node_failed.py

```diff
@@ -50,7 +50,12 @@ def make_key(site, node_type, datastore, etcd_key):
   error_syncer = EtcdSynchronizer(NullPlugin(key), dead_node_ip, etcd_ip=etcd_ip, force_leave=True)
 
 # Check that the dead node is even a member of the cluster
-etcd_result, idx = error_syncer.read_from_etcd(wait=False)
+etcd_result, idx = error_syncer.read_from_etcd(wait=False, timeout=10)
+
+if etcd_result is None:
+    print "ERROR:  Failed to contact etcd cluster on '{}' - node not removed".format(etcd_ip)
```

Comment:
I'm not a fan of this error message - it's not consistent with the other messages printed by this script. I think just removing the 'ERROR:  ' would be OK.